### PR TITLE
feat: new issues

### PR DESCRIPTION
**Pull request by rayanii**

### CODE REVIEW

**Comment by HammyPig** [comment]
very cool

**Review by HammyPig — COMMENTED** [review]
very cool

File: issues.txt [hunk 1]
```diff
@@ -105,4 +142,8 @@ Maximum amount of cards in hand
 	5 because 7 is too much
 
 
-Word for reshuffle
```
Comment (HammyPig) [inline issues.txt:108]: this was a great change thank you

File: issues.txt [hunk 2]
```diff
@@ -83,10 +110,20 @@ Feeling 'stuck' with cards
 	Army is now classed as a building:
 		No longer:
 			Corrupt: Steal an army from play
-			Lifesteal: Destroy an army, heal yourself by 1
```
Comment (HammyPig) [inline issues.txt:86]: where did lifesteal go

File: issues.txt [hunk 3]
```diff
@@ -27,6 +36,24 @@ Picking up cards, beginning or end of turn?
 		People can't guess what is in your hand, because you always have 5 cards.
 		Exchanges seem better as everyone has 5 cards to exchange
 
+Hand with redundant cards
```
Comment (HammyPig) [inline issues.txt:39]: yeppers

File: issues.txt [hunk 4]
```diff
@@ -27,6 +36,24 @@ Picking up cards, beginning or end of turn?
...
+	
+	- You can choose to reshuffle, but you have to discard your entire hand.
+		Yes 
+			-Because you have to make a 'choice' whether to sacrifice your entire hand for a new unknown hand (that may be better or worse).
```
Comment (HammyPig) [inline issues.txt:43]: lookin good

File: issues.txt [hunk 5]
```diff
@@ -27,6 +36,24 @@ Picking up cards, beginning or end of turn?
...
+			-Adds more strategical thinking.
+		No
+			-Adds more rules - overcomplicates.
+				-Solution - Quote the rule, without 'explaining' it.
+		
+	- You can choose to put down the cards, you don't want.
+		Yes
+			- Add more strategical thinking.
```
Comment (HammyPig) [inline issues.txt:51]: okay thats simply amazing

File: issues.txt [hunk 6]
```diff
@@ -27,6 +36,24 @@ Picking up cards, beginning or end of turn?
...
+				How? Because it reveals something to the other players about hand (for example, if a certain player is refusing to put down certain cards.
+					Redundant because you can still achieve the same goal with the previous solution, if a person refuses to reshuffle their hand,
+					regardless of not being able to make any immediate plays.
+	In summary, it's flexibility vs strategy. The strategy is that is it worth keeping a good card or should you free your hand?
+
```
Comment (HammyPig) [inline issues.txt:56]: not very good

**Review by HammyPig — COMMENTED** [review]
just a few words

File: issues.txt (hunk 2 above)
Comment (rayanii) [reply issues.txt:86]: lifesteal still there we are not changing it

**Pull request merged**